tests/basic/storage_pool_test.go

```diff
@@ -6590,9 +6585,16 @@ var _ = Describe("{PoolResizeInvalidPoolID}", func() {
 
 			log.InfoD("Current Size of the pool [%s] is [%d]", poolUUID, poolToBeResized.TotalSize/units.GiB)
 
+			alertType := api.SdkStoragePool_RESIZE_TYPE_AUTO
 			// Now trying to Expand Pool with Invalid Pool UUID
 			err = Inst().V.ExpandPoolUsingPxctlCmd(*nodeDetail, invalidPoolUUID,
-				api.SdkStoragePool_RESIZE_TYPE_AUTO, expectedSize, false)
+				alertType, expectedSize, false)
+			if err != nil {
+				if strings.Contains(fmt.Sprintf("%v", err), "Please re-issue expand with force") {
```

Comment:
handle the case where err!=nil and error does not contain Please re-issue expand with force